client/my-sites/plans/navigation.jsx

```diff
@@ -42,7 +42,7 @@ class PlansNavigation extends Component {
 		const { site, shouldShowMyPlan, shouldShowPlans, translate, isFreeOrFlexible } = this.props;
 		const path = sectionify( this.props.path );
 		const sectionTitle = this.getSectionTitle( path );
-		const hasPinnedItems = isMobile() && site;
+		const hasPinnedItems = Boolean( site ) && isMobile();
```

Comment:
Another PropType warning fix. Prior to the fix, when on mobile viewport, the value of `hasPinnedItems` was equal to the `site` object, when it is supposed to be a boolean.